user_tools/src/spark_rapids_pytools/common/utilities.py

```diff
@@ -78,7 +78,9 @@ def get_base_release(cls) -> str:
         # make sure that we replace micro version with 0
         version_comp = list(version_tuple)
         version_comp[2] = 0
-        res = '.'.join(str(v_comp) for v_comp in version_comp[:3])
+        # For now, tools jar is under url YY.MM.0 where MM is 02, 04, 06, 08, 10, and 12
+        # res = '.'.join(f'{v_comp:02}' for v_comp in version_comp[:3])
+        res = f'{version_comp[0]}.{version_comp[1]:02}.0'
```

Comment:
we will be having minor version supported very soon (e.g. 23.02.01).  should we add support for that now?

Reply:
Does the minor release include the Jar too? for example, the jar can be https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12/23.02.01/rapids-4-spark-tools_2.12-23.02.01.jar ?

Reply:
Yes, we will start launching minor versions for the tools jar as well as the user tools package in the 23.04 timeframe once we have automated releases fully ready.

Reply:
I updated the description of the PR to reflect the new behavior.

Example:
if the release is defined as 23.02.1
the wrapper will parse the content of https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12 to find that 22.12 is the highest version.
so, it will be used as default jar.